src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -34,6 +37,9 @@ internal abstract class AbstractGoToDefinitionService : AbstractFindDefinitionSe
             _streamingPresenter = streamingPresenter;
         }
 
+        async Task<IEnumerable<INavigableItem>> IGoToDefinitionService.FindDefinitionsAsync(Document document, int position, CancellationToken cancellationToken)
+            => await FindDefinitionsAsync(document, position, cancellationToken).ConfigureAwait(false);
```

Comment:
Note: classes can't be covariant, so we have to actually await to observe the `ImmutableArray`->`IEnumerable` implicit conversion.

Reply:
yup.  that's normal for us.